src/frontends/jax/include/openvino/frontend/jax/decoder.hpp

```diff
@@ -64,6 +65,8 @@ class JaxDecoder : public IDecoder {
     /// If there is no query for specific sub-graph it shouldn't be converted
     // node_visitor is a function that will be fed by nodes in subgraph for all nodes in graph
     virtual void visit_subgraph(std::function<void(std::shared_ptr<JaxDecoder>)> node_visitor) const = 0;
+
+    ~JaxDecoder() override;
```

Comment:
let us make all destructors virtual

Reply:
when base class is virtual, derived class automatically has virtual dtor